distribution/lib/Standard/Base/0.0.0-dev/src/Metadata.enso

```diff
@@ -85,3 +85,8 @@ make_single_choice values display=Display.Always =
         _ : Vector -> Choice.Option value.first value.second
         _ : Text -> Choice.Option value value.pretty
     Widget.Single_Choice (values.map make_option) Nothing display
+
+## PRIVATE
+make_delimiter_selector : Widget
+make_delimiter_selector =
+    make_single_choice [',', ';', '|', ['{tab}', "'\t'"], ['{space}', "' '"], ['{newline}', "['\n', '\r\n', '\r']"], ['Custom', ""]]
```

Comment:
I'm not sure if this file is the right place for this method?

The `Widget` file is our base interface, `make_single_choice` is a helper tied pretty closely to it.

But `make_delimiter_selector` seems to be a much more 'concrete' implementation.

Maybe we should have some files for such common widgets? They usually come as a helper on the related class. Since here the related class is a very fundamental one (`Text`), I'm indeed not sure if we want to pollute it. Maybe we just need to have some separate file for it like `Common_Widgets`?

Reply:
(But I totally appreciate reducing the logic duplication here)